### DISCUSSION TITLE
setting up original date export

### DISCUSSION THREAD

**Discussion by tomislavmedak** [retorquere/zotero-better-bibtex#2006] [Q&A] [answered]
I'm struggling to get the original date entry exported to my .bib file. I've entered {:original-date: YYYY} in my extra field and when I create the bibliography from Zotero using a CSL file that supports original-date, it works as advertised. However, BBT doesn't create an origial date entry for that item in the .bib file.

Do I need to add something in the postscript?

(My BBT is v5.6.10, Zotero is v5.0.96.3)

Thanks in advance for help.

**Comment by retorquere** [comment, marked as answer]
try exporting as Better BibLaTeX rather than Better BibTeX. bibtex has no support for origdate.

**Comment by tomislavmedak** [comment]
Works like a charm, thanks! Took me a bit of further figuring out that I can use [origyear] in citation key instead of [year] and that items with no original year entry will have their year of publication included instead, which is a nice bonus.
